Mathlib/Analysis/Complex/UpperHalfPlane/MoebiusAction.lean

```diff
@@ -160,49 +220,69 @@ theorem c_mul_im_sq_le_normSq_denom : (g 1 0 * z.im) ^ 2 ≤ Complex.normSq (den
 @[simp]
 theorem neg_smul : -g • z = g • z := by
   ext1
-  change _ / _ = _ / _
-  field_simp [denom_ne_zero]
-  simp only [num, denom, Complex.ofReal_neg, neg_mul, GLPos.coe_neg_GL, Units.val_neg, neg_apply]
-  ring_nf
+  simp [coe_smul, (show num (-g) z = -(num g z) by simp [num]; ring),
```

Comment:
`(show num (-g) z = -(num g z) by simp [num]` maybe this should be its own simp lemma, same with denom.